pkg/minikube/cluster/cluster.go

```diff
@@ -386,9 +386,12 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	}
 
 	if config.ShellProxyEnv != "" {
+		print("-- Setting proxy information ... ")
```

Comment:
It will even result in a writing to `stderr`; missed during the integration fix.